### PR TITLE
Add bfd variables

### PR DESCRIPTION
**Pull request by archerhungbsn**
Reviewer: @kenchiang @poolakiran

### CODE REVIEW

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/PR-620-b2

File: openflow_input/bsn_tlv [hunk 1]
```diff
@@ -1249,3 +1249,127 @@ struct of_bsn_tlv_passive : of_bsn_tlv {
...
+    uint16_t type == 182;
+    uint16_t length;
+    uint32_t value;
+};
+
+struct of_bsn_tlv_bfd_local_multiplier : of_bsn_tlv {
+    uint16_t type == 183;
+    uint16_t length;
+    uint8_t value;
+};
```
Comment (kenchiang) [inline openflow_input/bsn_tlv]: let's make this generic, i.e. of_bsn_tlv_multiplier, u32 value

File: openflow_input/bsn_tlv [hunk 2]
```diff
@@ -1249,3 +1249,127 @@ struct of_bsn_tlv_passive : of_bsn_tlv {
...
+    OF_BSN_BFD_ENDPOINT_EVENT_STATE_INIT = 2,
+    OF_BSN_BFD_ENDPOINT_EVENT_STATE_UP = 3,
+    OF_BSN_BFD_ENDPOINT_EVENT_SESSION_ERROR = 4,
+};
+
+struct of_bsn_tlv_bfd_endpoint_event : of_bsn_tlv {
+    uint16_t type == 188;
+    uint16_t length;
+    enum ofp_bsn_bfd_endpoint_event value;
+};
```
Comment (kenchiang) [inline openflow_input/bsn_tlv]: let's remove this, and use of_bsn_tlv_bfd_state (as defined above)

File: openflow_input/bsn_tlv [hunk 3]
```diff
@@ -1249,3 +1249,54 @@ struct of_bsn_tlv_passive : of_bsn_tlv {
...
+struct of_bsn_tlv_identifier : of_bsn_tlv {
+    uint16_t type == 173;
+    uint16_t length;
+    uint32_t value;
+};
+
+struct of_bsn_tlv_multiplier : of_bsn_tlv {
+    uint16_t type == 174;
+    uint16_t length;
+    uint8_t value;
```
Comment (kenchiang) [inline openflow_input/bsn_tlv]: let's make the value a uint32_t

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/PR-620-b3

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/PR-620-b4

**Comment by kenchiang** [comment]
ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  https://jenkins.bigswitch.com/job/loxigen-abat-automerge/11015//console

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
https://jenkins.bigswitch.com/job/loxigen-abat-automerge/11015//console
/cc

**Pull request merged**